cityofaustin/atd-moped
# Ability to create a status update from the Comments tab

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/14256

## Testing
**URL to test:** 
https://deploy-preview-1166--atd-moped-main.netlify.app/moped/ 

**Steps to test:**
1. Go to the "Notes" tab on a project (formerly called "Comments").
2. Toggle between the "Internal Note" and "Status Update" options above the Save button for a new note. Test saving both types of new notes.
3. Check that you see your new note pop up in the notes section under the correct note type toggle.
4. After adding a new status update, check on the project summary tab that it is reflected there in the Status update section as well.
5. Test editing a comment, in editing state you should also see the radio buttons and be able to update the note type.
6. Also test sizing down your screen to see the way the buttons change. I tested on my iPhone and they look fine, lmk if you want them to look differently on a smaller sized screen!


---
#### Ship list
